src/editing/createfeatureComponent.js

```diff
@@ -239,6 +239,14 @@ exports.Controller_.prototype.$onInit = function() {
   this.interaction_ = interaction;
   this.map.addInteraction(interaction);
 
+  this.keyDownListener_ = document.body.addEventListener('keydown', (evt) => {
```

Comment:
It won't work: `addEventListener` doesn't return anything.

Use something like this instead:
```js
 this.ngeoEventHelper_.addListenerKey(
      uid,
      olEvents.listen(
        document.body,
        'keydown',
        this.handleKeyDown_,
        this
      )
    );
```